Dockerfile

```diff
@@ -1,38 +1,41 @@
-FROM php:8.3.4-cli-alpine3.19 AS build
+FROM php:8.3.10-cli-alpine3.20 AS build
 
-RUN apk update && \
-  apk add --no-cache ca-certificates curl jo zip unzip
+RUN apk update \
+    && apk add --no-cache ca-certificates curl jo zip unzip
 
 WORKDIR /usr/local/bin
 
-RUN curl -L -o install-php-extensions https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions && \
-  chmod +x install-php-extensions && \
-  install-php-extensions ds-1.5.0 intl
+RUN curl -L -o install-php-extensions \
+    https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions \
+    && chmod +x install-php-extensions \
+    && install-php-extensions intl
```

Comment:
some student solutions use intl, also ds (which has been removed in this change)

Reply:
@neenjaw Is there a way to find the solutions using `ds`?

Reply:
Not to my knowledge

Reply:
@neenjaw I'll add `ds` again, then. I thought it was only used for `junit-handler` itself.

@homersimpsons `intl` provides the functions to handle Unicode normalization (`Normalizer` class) and grapheme handling (`grapheme_*` string functions similar to `mb_*`). It is pretty much the only way to ever have proper Unicode grapheme handling. Oh, and it can handle IDN domain name conversion. If we want to add a concept for that.

Reply:
Ok, so those extensions are for students solutions. Maybe add a comment about this here?

Also I do not know if they are mentioned in the doc (I didn't search) on my side I thought there were no extensions.

Reply:
See #132 for a follow-up.